server/repository/src/migrations/v1_08_00/assets/mod.rs

```diff
@@ -1,8 +1,10 @@
-use crate::StorageConnection;
+use crate::{assets::asset_category, StorageConnection};
 
+pub mod asset_catalogue;
 pub mod reference_data;
 
 pub(crate) fn migrate_assets(connection: &StorageConnection) -> anyhow::Result<()> {
     reference_data::migrate(connection)?;
+    asset_catalogue::migrate(connection)?;
```

Comment:
technically this is reference data as well.. but I thought it was nice to have in its own file

Reply:
I think the others are fine to have together